yash-datalake/nested/blobTriggerdFunction.json

```diff
@@ -0,0 +1,275 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    },
+    "kibanaFunctionHost": {
+      "type": "String"
+    },
+    "kibanaFunctionPath": {
+      "type": "String"
+    },
+    "EsIP": {
+      "type": "String"
+    },
+    "serverfarms_EastUS2Plan_name": {
+      "type": "String"
+    },
+    "sites_demotriggerapp1234_name": {
+      "type": "String"
+    },
+    "config_web_name_4": {
+      "type": "String"
+    },
+    "storage_name": {
+      "type": "string"
+    }
+  },
+  "variables": {
+    "pythonExtension": "azureappservice-python2714x64",
+    "raweventGridSubscriptionName": "rawtopicsubscription1a",

```

Comment:
you need to account for uniqueness requirements in your resource names

Reply:
Made them unique according to best practices